_pages/ato/ssp.md

```diff
@@ -6,19 +6,25 @@ As described in [the NIST guide](http://csrc.nist.gov/publications/nistpubs/800-
 
 > The purpose of the system security plan is to provide an overview of the security requirements of the system and describe the controls in place or planned for meeting those requirements.
 
-18F uses a machine-readable format for which template is below. We also have a tool to help generate SSPs, called [Compliance Masonry](https://github.com/opencontrol/compliance-masonry).
+The system security plan (SSP) is a long Google Doc. 18F is actively working on a tool to generate them called [Compliance Masonry](https://github.com/opencontrol/compliance-masonry), which uses a machine-readable format for which template is below. For now, the Google Docs are the canonical versions of the SSP.
 
 ### System/Network Diagrams
 
-One of the requirements for an SSP is to include a network diagram for your system. A few example diagrams can be found below:
+One of the requirements for an SSP is to include a network diagram for your system. It is helpful to create this diagram as a Google Drawing so that your Infrastructure Lead and AO can edit it as needed during the ATO assessment process.
 
-1. [A simple application running on cloud.gov](https://docs.google.com/drawings/d/1f7y1nAkNfFX0MuHNB11z6bNCoNohQJ5f4s4xLr4zxds/edit)
-1. [A more complex application running on cloud.gov](https://docs.google.com/drawings/d/1LomjFFPlpIrO4i5H13ECJFVLCQ2i-nvJhiyd6IyUQYg/edit)
+The diagram should be as detailed as possible. Include things like what ports are in use, and what they are used for. 
+
+When creating the diagram include the "ATO Boundary." A dotted line box is a nice way to do this. The system diagram includes things outside of the ATO boundary for context, such as the cloud.gov platform itself, and delineating the parts of the diagram being ATOd versus the parts that exist for context is important.
+
+A few example diagrams can be found below:
+
+1. [A simple application running on cloud.gov](https://docs.google.com/drawings/d/1nwclBJQfbuzsnGOqe88VukQl3uiH1Jfa4c0FT1Cq43I/edit)
+1. [A more complex application running on cloud.gov](https://docs.google.com/drawings/d/1k1wykk5PbLKSNJj8FyZbIlpX0D8r1q3-w-uRK_WWt9g/edit)
 1. [A complex application not running on cloud.gov](https://docs.google.com/drawings/d/10cH-OUB1NWzCI0v9LPzm7AXCfrHXNkDgnae-7hcUFu8/edit)
 
-### Template
+### Compliance Masonry Template
```

Comment:
The `system-security-plan.yml` doesn't actually have anything to do with Masonry (for now, at least).